resolvers/donationResolver.ts

```diff
@@ -144,8 +144,14 @@ export class DonationResolver {
       const baseTokens =
         Number(priceChainId) === 1 ? ['USDT', 'ETH'] : ['WXDAI', 'WETH']
 
+      let donorEmail = ""
```

Comment:
Would be more elegant like:
`let donorEmail = (user != null) ? user.email : ''`